### ISSUE TITLE
Notes disappeared after upgrade to version 0.11.12 and new note with the same title does not save data

### ISSUE THREAD

**Issue by ivansharamok** [BoostIO/Boostnote#2727]
# Current behavior

After upgrading Boostnote to version `0.11.12`, a few of my notes disappeared. I use Google Drive as storage device for my notes. I tried removing storage from Boostnote configuration and then adding it back but that didn't bring the missing notes back.

Since I wasn't able to recover the notes, I created a new note with the same title as missing one. However, when I record anything into that note, the data does not get preserved. Once I navigate away from the note, the data gets wiped out from it.

# Expected behavior

Notes should not disappear when upgrading Boostnote.
The note data should not disappear when one navigates away from the note.

# Steps to reproduce

Since the issue started after the upgrade to `0.11.12` version, I don't have any steps to repro the problem besides upgrading to version `0.11.12`.

For the data disappearance here are my steps:
1.  Create a new note with a title (e.g. `# Caption`)
2. Type in a few lines of text into the note and navigate away form it (choose another note)
3. Return back to your note with `# Caption`.

## Result 
In my case the data is gone except for `# Caption` line. Once I restart Boostnote, my note with `# Caption` title disappears.

# Environment

- Version : `0.11.12`
- OS Version and name : `macOS Mojave Version 10.14.1 (18B75)`

**Comment by ZeroX-DG** [comment]
This sound like a critical bug. @Rokt33r can you reproduce the issue on mac?

**Comment by ivansharamok** [comment]
I removed my note with `# Caption` title and restarted Boostnote. I can no longer repro the issue with content disappearance. 
I use GDrive as my storage device and sometimes edit my notes when I don't have Internet, could such scenario cause issues with data sync? Is there a temp local storage for Boostnote that I could check for missing notes?

**Comment by ivansharamok** [comment]
unfortunately I didn't take a note of version that I used before upgrade :(. The scenario in which Boostnote wouldn't save changes to backing storage (i.e. GDrive) is possible but I recall restarting Boostnote a few times as I kept updating those notes over the period of 3 months or so.

It would be nice to have some sort of warning/banner that would warn a user in case Boostnote couldn't push note changes to the storage device.
I'm closing this issue as I can no longer repro it.
